### PR TITLE
send_email functionality in utils.email_service

### LINKED ISSUE #52: Setup Email service utils
**Is your feature request related to a problem? Please describe.**
For notification purposes we require to send emails. So we a email service setup.

**Describe the solution you'd like**
1. Create a folder utils under project root
1. create utils.email_service.send_email(recipient: str, subject: str: body: str) 
1. create the necessary setup that when this function is called the email is sent.
1. Update readme file.

### PR DESCRIPTION
**Pull request by anshsrtv**
### Description
As required, the utils folder is updated with a new file named email_service.py which can be imported in any of the project file to send emails asynchronously. Make sure the celery and redis-server are up and running while testing this. Also update the .env file with the email and password of any gmail account.

**Type of Change:** (Delete irrelevant options)
- Utilities of the application updated allowing emails to be sent easily and asynchronously.
- server/settings changed to include EMAIL_SETTINGS and CELERY_IMPORTS added.

## How Has This Been Tested?
This has been tested using the python shell by importing the send_email from utils.email_service. The functionality works in sync as well as in async. 

### Checklist:
-  My PR follows the style guidelines of this project
-  I have performed a self-review of my own code or materials
- Update Readme.md

### CODE REVIEW

File: utils/email_service.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from celery import shared_task
+from django.core.mail import send_mail
+from decouple import config
+
+@shared_task
+def send_email(recipient,subject,body):
+    send_mail(subject, body, config('EMAIL_HOST_USER') , [recipient,])
```
Comment (naveennvrgup) [inline utils/email_service.py]:
```suggestion
    send_mail(subject, body, config('EMAIL_HOST_USER') , recipient)
```

let the users enter a list of recipients

Comment (naveennvrgup) [reply utils/email_service.py]: When i try to send html templated mail it doesn't work. It send the mail as text. 

image

Please try something like this. We will be send html email only. :innocent: 

image

**Review by naveennvrgup — CHANGES_REQUESTED** [review]
1. Great job with your first PR. :fire: @anshsrtv 
1. Please address my comments.
1. Dont worry about travis checks failing we have not added the new env var to it.
1. Please work from forked repo's. I will provide you further instruction in today meeting.

**Comment by naveennvrgup** [comment]
@anshsrtv Instead of `Fixes #ISSUE_NUMBER #52` do `Fixes #52`

**Comment by naveennvrgup** [comment]
Please dont mark the comments as resolved by your own it has to be resolved by the person who commented it :innocent:

File: utils/email_service.py (hunk 1 above)
Comment (anshsrtv) [reply utils/email_service.py]: Made the changes, please check again.

**Review by naveennvrgup — APPROVED** [review]
1. Tested locally work great. :fire: 
1. Congratulations on your first PR @anshsrtv  :confetti_ball:
1. Merging PR

**Pull request merged**